css/main.css

```diff
@@ -487,3 +487,66 @@ footer {
     margin: 40px 0;
   }
 }
+/* for Art Feature */
+
+.welcome-title {
+  margin-top: 20px;
+  font-size: 2.75rem;
+  text-align: center;
+}
+.editor {
+  margin-top: 70px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.html-container {
+  border: 1px solid grey;
+  border-radius: 5px;
+  padding: 30px;
+  margin-right: 20px;
+  width: 300px;
+  height: 325px;
+  display: flex;
+  flex-direction: column;
+}
+#html-code {
+  width: 300px;
+  height: 300px;
+  margin-top: 10px;
+}
+.output-container {
+  border: 1px solid grey;
+  border-radius: 5px;
+  padding: 30px;
+  width: 300px;
+  height: 325px;
+}
+
+#output {
+  width: 300px;
+  height: 328px;
+}
+input {
+  text-align: center;
+  border: none;
+  outline: none;
```

Comment:
Outline: none is not advisable considering accessibility.
I will share you a link in discord channel, you can refer that and could try making relevant changes

Reply:
https://betterprogramming.pub/a11y-never-remove-the-outlines-ee4efc7a9968